ppanggolin/annotate/annotate.py

```diff
@@ -327,16 +327,22 @@ def readAnnotations(pangenome, organisms_file, getSeq = True, pseudo = False):
     pangenome.parameters["annotation"] = {}
     pangenome.parameters["annotation"]["read_annotations_from_file"] = True
 
-def getGeneSequencesFromFastas(pangenome, fasta_file):
+def getGeneSequencesFromFastas(pangenome, fastas, fastaList=True):
     fastaDict = {}
-    for line in read_compressed_or_not(fasta_file):
-        elements = [el.strip() for el in line.split("\t")]
-        if len(elements)<=1:
-            logging.getLogger().error("No tabulation separator found in organisms file")
-            exit(1)
-        org = pangenome.addOrganism(elements[0])
-        with read_compressed_or_not(elements[1]) as currFastaFile:
-            fastaDict[org] = read_fasta(org, currFastaFile)
+    if fastaList:
+        for line in read_compressed_or_not(fastas):
+            elements = [el.strip() for el in line.split("\t")]
+            if len(elements)<=1:
+                logging.getLogger().error("No tabulation separator found in organisms file")
+                exit(1)
+            org = pangenome.addOrganism(elements[0])
+            with read_compressed_or_not(elements[1]) as currFastaFile:
+                fastaDict[org] = read_fasta(org, currFastaFile)
+    else:
+        for fasta in fastas:
+            fasta = pangenome.addOrganism(fasta)
```

Comment:
Maybe remove file extentions for organism names ?

Reply:
Although it can be dirty, the organism names must not be modified by the program in my opinion. I think this is the user responsibility to properly name its fasta files.
In MASH for example the raw file path is used as genome identifier.

Reply:
Alright, fine by me